src/Microsoft.ML.AutoMLFeaturizers/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using Microsoft.ML;
+
+[assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.Tests" + PublicKey.TestValue)]
```

Comment:
What internals are the tests using? I think you should be able to test everything from the public API, right?

Reply:
I was testing the constructor that took in the `Options` class, but looking at the other Transformers and their tests they are going about it differently. Let me fix it and remove that `VisibleTo` tag.

Reply:
So I just double checked, and I was testing the entry point that AutoML uses and it takes in the `Options` class which is internal. Is it ok to leave everything as is?

Reply:
Yeah, all our tests have IVT, so this is fine to leave. I was mostly just curious why it was needed here. IMO it is best to test without Internals Visible To, but if you need it don't worry.